charts/jaeger/templates/query-ing.yaml

```diff
@@ -2,7 +2,7 @@
 {{- if .Values.query.ingress.enabled -}}
 {{- $servicePort := .Values.query.service.port -}}
 {{- $basePath := .Values.query.basePath -}}
-{{- if $useNetworkingV1 }}
+{{- if semverCompare ">=1.19" .Capabilities.KubeVersion.Version -}}
```

Comment:
What is wrong with this check?
```
{{- $useNetworkingV1 := .Capabilities.APIVersions.Has "networking.k8s.io/v1" -}}
```

Reply:
It might be related to a bug in `helm` https://github.com/databus23/helm-diff/issues/165
Which version of helm are you using? Can you try upgrading and see if you'd have the same issue?
I'd rather not use version comparisons. We should always check if the appropriate API version is available regardless of the Kubernetes version being used.